docs/Encyclopedia/pages/Birdseed.md

```diff
@@ -0,0 +1,32 @@
+# Birdseed
+
+## Description ##
+
+Birdseed is a genotyping algorithm used for the detection of single nucleotide polymorphisms (SNPs) in GDC genotyping array data harmonization.
+
+## Overview ##
+
+Birdseed is utilized for genotyping SNPs from microarray data. This pipeline processes raw array data, identifies SNPs, and generates genotype calls. It is specifically designed for genotyping arrays such as the Affymetrix SNP 6.0 array, providing accurate SNP detection across different sample types.
+
```

Comment:
Add this:  "Birdseed files were originally processed and generated by the TCGA program and brought into the GDC Data Portal when the GDC Legacy Archive was retired."

Reply:
added

Reply:
thank you!!